### ISSUE TITLE
LMS support

### ISSUE THREAD

**Issue by viandanteoscuro** [ajay2507/ebay-node-api#35]
Thankyou for your work!

Is there a LMS support?
Thanks

**Comment by ajay2507** [comment]
@viandanteoscuro Nope. I can take that as improvement. If you are interested in specific calls, I can implement that.
https://developer.ebay.com/DevZone/large-merchant-services/Concepts/LandAPIRef.html

If you want, feel free to contribute.
